### DISCUSSION TITLE
Remote Play together possible?

### DISCUSSION THREAD

**Discussion by toby63** [LizardByte/community#196] [Sunshine] [answered]
This might be a silly question, but I could not find a complete or understandable answer to this yet.

On Steam you can remote play together, which (as you probably know) means that the host and client play together in one game.

Is this possible with Sunshine and Moonlight?

I was a bit confused by the answers given in LizardByte/community#44 and LizardByte/community#120 (those seem to cover cases of multiple clients playing together).
So maybe someone could explain that a little more.

**Comment by ReenigneArcher** [comment, marked as answer]
See here: https://docs.lizardbyte.dev/projects/sunshine/en/latest/about/advanced_usage.html#channels

**Reply by toby63** [reply]
Thats exactly the problem, I don't quite understand what that is used for.

Because why do I need distinct video streams, if the concept of remote play together is to have the same screen?
And why would I need multicast, if I only have one client?

**Reply by ReenigneArcher** [reply]
> And why would I need multicast, if I only have one client?

To remote play with more than one person on more than one device would be more than one client.

Each client likely needs a separate channel unless they are identical clients. It's to encode the video according to the parameters of the moonlight client app for each person. It still streams the same content, but with different parameters.

**Reply by toby63** [reply]
> To remote play with more than one person on more than one device would be more than one client.

So even if the host is one of two participants, I would need this option?
Or how would this work exactly?

**Reply by ReenigneArcher** [reply]
I think you're misunderstanding what a host and client is. The host is the server that runs Sunshine. The client is the device that runs moonlight.

If you want to stream Sunshine to two clients at the same time, you need to increase the "channels" setting. I don't know how else to explain it.

**Reply by toby63** [reply]
I understand that.

As I asked above:
> On Steam you can remote play together, which (as you probably know) means that the host and client play together in one game.

So in that case the host is a computer which runs the game and is also directly used by one of two players.
The second player of course uses a different computer and connects as a client.

My question was and is whether such a scenario is possible? (I haven't used Sunshine/Moonlight yet)
Because I have only read about multiple clients connecting to a seperate server.

**Reply by ReenigneArcher** [reply]
> My question was and is whether such a scenario is possible?

Yes, IF you set `channels` to 2.

**Reply by toby63** [reply]
Ok, thx for clarification. I will try it out then.
